apps/storybook/src/Table.stories.tsx

```diff
@@ -2953,7 +2953,37 @@ export const ColumnManager: Story<Partial<TableProps>> = (args) => {
               );
             },
           },
-          ActionColumn({ columnManager: true }),
+          {
+            id: 'Price',
+            Header: 'Price',
+            accessor: 'price',
+          },
+          {
+            id: 'Color',
+            Header: 'Color',
+            accessor: 'color',
+          },
+          {
+            id: '# in stock',
+            Header: '# in stock',
+            accessor: 'stock',
+          },
+          {
+            id: 'Rating',
+            Header: 'Rating',
+            accessor: 'rating',
+          },
+          {
+            id: 'Location',
+            Header: 'Location',
+            accessor: 'location',
+          },
+          ActionColumn({
+            columnManager: {
+              className: '', // To customize styling of DropdownMenu in ActionColumn
```

Comment:
If `style` and `className` are optional, why are we passing blank parameters for them when we can just pass `columnManager: true`?

Reply:
I left that so that people going through the storybook code can know that they can pass an object too for `columnManager`  to customize the styles instead of just passing a boolean. But I could remove it too if it feels unnecessary

Reply:
I don't think it's necessary. Examples such as `DatePicker` and `DropdownMenu` can be given custom styling and there are unit tests to check for that, but it's not stated anywhere in their stories.

Reply:
**[DONE]** Thanks